doc/source/qip-basics.rst

```diff
@@ -272,31 +272,136 @@ gate function returning a :class:`qutip.Qobj` and save it in the attribute ``use
 
 .. _quantum_circuit_plots:
 
-Plotting a Quantum Circuit
+Plotting Quantum Circuits
 ===================================
 
-A quantum circuit (described above) can directly be plotted using the QCircuit library (https://github.com/CQuIC/qcircuit).
-QCiruit is a quantum circuit drawing application and is implemented directly into QuTiP.
-
-More information related to installing these packages is also available in the
-installation guide (:ref:`circuit_plot_packages`).
-
-An example code for plotting the example quantum circuit from above is given:
+QuTiP-QIP offers three distinct methods for visualizing quantum circuits. Below is an example that demonstrates how to create and plot a quantum circuit using these methods:
 
 .. code-block:: python
 
     from qutip_qip.circuit import QubitCircuit
     from qutip_qip.operations import Gate
+
     # create the quantum circuit
     qc = QubitCircuit(2, num_cbits=1)
     qc.add_gate("CNOT", controls=0, targets=1)
     qc.add_gate("SNOT", targets=1)
     qc.add_gate("ISWAP", targets=[0,1])
     qc.add_measurement("M0", targets=1, classical_store=0)
-    # plot the quantum circuit
-    qc.png
 
-.. image:: /figures//quantum_circuit_example.png
+- **Matplotlib (Default)**:
+
+    .. code-block:: python
+          
+        # plot the quantum circuit using  MatRenderer
+        qc.draw("matplotlib")
+
+    .. image:: /figures//mat_circuit_example.png
```

Comment:
It is better to use the plot derivative than upload the `png` here. We used it for latex because there is really no other way.

https://github.com/qutip/qutip-qip/blob/master/doc/source/qip-processor.rst#spinchain

It needs to be rebased to remove png from the history.

Reply:
You can download and view the rendered `html` here in the artifact.

https://github.com/qutip/qutip-qip/actions/runs/10496547795?pr=244